Oqtane.Shared/Shared/Utilities.cs

```diff
@@ -217,5 +219,18 @@ private static string RemapInternationalCharToAscii(char c)
                 return "";
             }
         }
+
+        public static bool IsValidEmail(string email)
+        {
+            if (email != "")
+            {
```

Comment:
```c#
if(string.IsNullOrEmpty(email)) return false; 
```
should be here i think.

Reply:
I kept the logic the same from the razor file because that's what you said to do. I will make the change. I would have done it like this too, but I was following your instructions and others.

Reply:
@chlupac yes I did test it.